components/Navigation.js

```diff
@@ -100,6 +100,7 @@ export default class Navigation extends React.PureComponent {
             <List>
                 <Lines color="rgba(0, 0, 0, 0.07)" />
                 <ul>
+                    <li><a href="https://blog.agent.sh">Blog</a></li>
```

Comment:
As a user I am not a big fan of websites which automatically open new windows, my tabs are already chaotic enough 😄 But if you think it makes sense I can still add it.